lib/datadog/profiling.rb

```diff
@@ -136,6 +136,7 @@ def self.allocation_count # rubocop:disable Lint/NestedMethodDefinition (On purp
       return false unless supported?
 
       require_relative 'profiling/ext/forking'
+      require_relative 'profiling/ext/dir_monkey_patches'
```

Comment:
I went with this name because I know it will show up in the profiling flamegraph for customers.

My thinking is that it's clearer if you see "oh, why is there a datadog thing here? ah, it's monkey patching my uses of dir". Signal masking makes me think a bit more along hte lines of "what is this signal masking thing? what is masking anyway?".

Reply:
Ha, interesting! I was thinking that if I saw that in a stack trace or something I would wonder "why is Datadog monkey patching dir stuff at all?". In a way "monkey patching" is how but not what/why. In a way it's also redundant with `ext`, the sort of conventional way to say "this folder contains things that extend stuff". `dir_signal_fix` maybe?

But at this stage that's becoming bikeshedding and certainly not a blocker. Feel free to move forward.

A comment with a short note at the top of the file _with links to upstream issues_ would go a long way though.

Reply:
I think the joke about naming things being hard applies here >_>. I suspect/fear just relying on `ext/` may be too subtle (e.g. when you're looking a flamegraph), but am open to changing if folks get confused about the current name.

+1 I've added a link to the upstream ticket in https://github.com/DataDog/dd-trace-rb/commit/259e134092eae57cb2ccbea028adc8e8d0fc7a19.